bot/exts/info/information.py

```diff
@@ -578,7 +589,12 @@ async def rules(self, ctx: Context, *args: Optional[str]) -> Optional[Set[int]]:
 
         return final_rule_numbers
 
+    async def cog_load(self) -> None:
+        """Carry out cog asynchronous initialisation."""
+        await self._set_rules_command_help()
+
 
 async def setup(bot: Bot) -> None:
     """Load the Information cog."""
     await bot.add_cog(Information(bot))
+    log.info("Cog loaded: Information")
```

Comment:
This is already done in a different place (here https://github.com/python-discord/bot-core/blob/db0ff48a574f5b466f3e8dc25cc32c3a9bfc1370/botcore/_bot.py#L155), so shouldn't be needed here.

Reply:
Oh, that's strange because I searched in the logs and I didn't find it.
Maybe because I was having issues in the first iteration.
Removed !